src/Features/Core/Portable/PullMemberUp/Dialog/PullMemberUpWithDialogCodeAction.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.  
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeRefactorings.PullMemberUp.Dialog;
+using Microsoft.CodeAnalysis.PullMemberUp;
+
+namespace Microsoft.CodeAnalysis.CodeRefactorings.PullMemberUp
+{
+    internal abstract partial class AbstractPullMemberUpRefactoringProvider
+    {
+        internal class PullMemberUpWithDialogCodeAction : CodeActionWithOptions
+        {
+            private readonly ISymbol _selectedMember;
+
+            private readonly Document _document;
+
+            private readonly IPullMemberUpOptionsService _service;
```

Comment:
nit: consider removing these blank lines between fields.

Reply:
@Cosifne FWIW, i can't resolve these comment.s  if you've actually resolved them, it would be nice to do that to keep the review cleaner :)